### DISCUSSION TITLE
Error accessing Database module

### DISCUSSION THREAD

**Discussion by gcsa99** [osm-search/Nominatim#2645] [Q&A]
I'm having problems trying to run the nominatim import command.

This is the log:
sislogica@srv-sislogica:~/nominatim-brazil$ nominatim import --osm-file /srv/osrm/osrm-run/brazil-latest.osm.pbf 2>&1 |tee setup.log
2022-03-21 14:48:24: Using project directory: /home/sislogica/nominatim-brazil
2022-03-21 14:48:25: Creating database
2022-03-21 14:48:32: Setting up country tables
2022-03-21 14:48:39: Importing OSM data file
2022-03-21 14:48:39  osm2pgsql version 1.5.1
2022-03-21 14:48:39  Database version: 13.6 (Ubuntu 13.6-1.pgdg21.10+1)
2022-03-21 14:48:39  PostGIS version: 3.2
2022-03-21 14:48:40  Parsing gazetteer style file '/usr/local/etc/nominatim/import-extratags.style'.
NOTICE:  table "place" does not exist, skipping
Processing: Node(229260k 106.3k/s) Way(16620k 9.61k/s) Relation(143720 1026.6/sProcessing: Node(229260k 106.3k/s) Way(16620k 9.61k/s) Relation(146700 1040.4/sProcessing: Node(229260k 106.3k/s) Way(16620k 9.61k/s) Relation(149730 1054.4/sProcessing: Node(229260k 106.3k/s) Way(16620k 9.61k/s) Relation(157960 1104.6/sProcessing: Node(229260k 106.3k/s) Way(16620k 9.61k/s) Relation(163020 1132.1/sProcessing: Node(229260k 106.3k/s) Way(16620k 9.61k/s) Relation(171740 1184.4/sProcessing: Node(229260k 106.3k/s) Way(16620k 9.61k/s) Relation(182000 1246.6/sProcessing: Node(229260k 106.3k/s) Way(16620k 9.61k/s) Relation(190180 1293.7/sProcessing: Node(229260k 106.3k/s) Way(16620k 9.61k/s) Relation(196570 1328.2/sProcessing: Node(229260k 106.3k/s) Way(16620k 9.61k/s) Relation(196640 1310.9/sProcessing: Node(229260k 106.3k/s) Way(16620k 9.61k/s) Relation(200530 1328.0/sProcessing: Node(229260k 106.3k/s) Way(16620k 9.61k/s) Relation(203410 1338.2/sProcessing: Node(229260k 106.3k/s) Way(16620k 9.61k/s) Relation(207590 1356.8/sProcessing: Node(229260k 106.3k/s) Way(16620k 9.61k/s) Relation(211940 1376.2/sProcessing: Node(229260k 106.3k/s) Way(16620k 9.61k/s) Relation(211990 1367.7/sProcessing: Node(229260k 106.3k/s) Way(16620k 9.61k/s) Relation(212020 1359.1/sProcessing: Node(229260k 106.3k/s) Way(16620k 9.61k/s) Relation(213450 1359.6/sProcessing: Node(229260k 106.3k/s) Way(16620k 9.61k/s) Relation(218110 1371.8/sProcessing: Node(229260k 106.3k/s) Way(16620k 9.61k/s) Relation(226710 1416.9/sProcessing: Node(229260k 106.3k/s) Way(16620k 9.61k/s) Relation(243030 1509.5/sProcessing: Node(229260k 106.3k/s) Way(16620k 9.61k/s) Relation(252300 1557.4/sProcessing: Node(229260k 106.3k/s) Way(16620k 9.61k/s) Relation(258130 1583.6/sProcessing: Node(229260k 106.3k/s) Way(16620k 9.61k/s) Relation(258230 1574.6/sProcessing: Node(229260k 106.3k/s) Way(16620k 9.61k/s) Relation(258270 1565.3/sProcessing: Node(229260k 106.3k/s) Way(16620k 9.61k/s) Relation(258650 1558.1/sProcessing: Node(229260k 106.3k/s) Way(16620k 9.61k/s) Relation(261960 1568.6/sProcessing: Node(229260k 106.3k/s) Way(16620k 9.61k/s) Relation(271140 1613.9/sProcessing: Node(229260k 106.3k/s) Way(16620k 9.61k/s) Relation(280620 1660.5/sProcessing: Node(229260k 106.3k/s) Way(16620k 9.61k/s) Relation(281760 1657.4/sProcessing: Node(229260k 106.3k/s) Way(16620k 9.61k/s) Relation(289200 1691.2/sProcessing: Node(229260k 106.3k/s) Way(16620k 9.61k/s) Relation(296290 1722.6/sProcessing: Node(229260k 106.3k/s) Way(16620k 9.61k/s) Relation(302070 1706.6/sProcessing: Node(229260k 106.3k/s) Way(16620k 9.61k/s) Relation(303230 1703.5/s                                                                               2022-03-21 15:57:42  Reading input files done in 4142s (1h 9m 2s).
2022-03-21 15:57:42    Processed 229260951 nodes in 2156s (35m 56s) - 106k/s
2022-03-21 15:57:42    Processed 16620971 ways in 1730s (28m 50s) - 10k/s
2022-03-21 15:57:42    Processed 304920 relations in 256s (4m 16s) - 1k/s
2022-03-21 15:57:47  Done postprocessing on table 'planet_osm_nodes' in 0s
2022-03-21 15:57:47  Done postprocessing on table 'planet_osm_ways' in 0s
2022-03-21 15:57:47  Done postprocessing on table 'planet_osm_rels' in 0s
2022-03-21 15:57:47  osm2pgsql took 4148s (1h 9m 8s) overall.
2022-03-21 15:57:47: Create functions (1st pass)
2022-03-21 15:57:49: Create tables
2022-03-21 15:58:05: Create functions (2nd pass)
2022-03-21 15:58:07: Create table triggers
2022-03-21 15:58:07: Create partition tables
2022-03-21 15:58:48: Create functions (3rd pass)
2022-03-21 15:58:49: Importing wikipedia importance data
2022-03-21 16:11:22: Initialise tables
2022-03-21 16:11:26: Load data into placex table
..................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................2022-03-21 16:36:07: Setting up tokenizer
2022-03-21 16:36:11: Error accessing database module: could not access file "/home/sislogica/nominatim-brazil/module/nominatim.so": Permission denied

2022-03-21 16:36:11: FATAL: Database module cannot be accessed.

*end log

$USERHOME: /srv/nominatim

What am I doing wrong?
I'm following exactly the official tutorial for the Ubuntu 20, but the server is running ubuntu 21.10 impish, is there any configuration change?.

**Comment by mtmail** [comment]
Check "Could not load library "nominatim.so"" on https://nominatim.org/release-docs/latest/admin/Faq/ It's possible the database user does not have access to a parent folder, e.g. `/home/sislogica/`
